deepchem/models/torch_models/torch_model.py

```diff
@@ -945,10 +953,10 @@ def default_generator(
     ([inputs], [outputs], [weights])
     """
     for epoch in range(epochs):
-      for (X_b, y_b, w_b, ids_b) in dataset.iterbatches(
-          batch_size=self.batch_size,
-          deterministic=deterministic,
-          pad_batches=pad_batches):
+      for (X_b, y_b, w_b,
+           ids_b) in dataset.iterbatches(batch_size=self.batch_size,
+                                         deterministic=deterministic,
+                                         pad_batches=pad_batches):
```

Comment:
Was this done by yapf? Doesn't seem right. I'm not sure if all these formatting changes are necessary.